### ISSUE TITLE
 INFO - duckdb.py:454 - No existing DB found in /root/localGPT/SOURCE_DOCUMENTS, skipping load  ？

### ISSUE THREAD

**Issue by shiliu2023** [PromtEngineer/localGPT#213]
why？

**Comment by aabalke33** [comment]
It looks like it thinks the database path is the source docs folder. The current standard implementation has a separate folder "/root/localGPT/DB" to store the database. Check if you have the database in the /DB folder. If you do, check the PERSIST_DIRECTORY in constants.py is the /DB folder. If these are not the problem please provide more information.

**Comment by Peterson-Repository** [comment]
I am having the Same Issue

duckdb.py:454 - No existing DB found in root\localGPT/DB, skipping load

**Comment by aabalke33** [comment]
I don't believe that error (454) means failure, just that it couldn't find a database and so it won't load one. It should then attempt to create one. Are there any following errors? If it looks held up it may be creating the database (this did take some time).

**Comment by anuragthakur77** [comment]
Yes it took me 7 mins
2023-07-29 16:06:00,887 - INFO - __init__.py:88 - Running Chroma using direct local API.
2023-07-29 16:06:12,145 - WARNING - __init__.py:43 - Using embedded DuckDB with persistence: data will be stored in: /Users/anurag.thakur/llama/localGPT/DB
2023-07-29 16:06:13,775 - INFO - ctypes.py:22 - Successfully imported ClickHouse Connect C data optimizations
2023-07-29 16:06:14,198 - INFO - json_impl.py:45 - Using python library for writing JSON byte strings
2023-07-29 16:06:14,874 - INFO - duckdb.py:454 - No existing DB found in /Users/anurag.thakur/llama/localGPT/DB, skipping load
2023-07-29 16:06:14,874 - INFO - duckdb.py:466 - No existing DB found in /Users/anurag.thakur/llama/localGPT/DB, skipping load


2023-07-29 16:12:37,512 - INFO - duckdb.py:414 - Persisting DB to disk, putting it in the save folder: /Users/anurag.thakur/llama/localGPT/DB
2023-07-29 16:12:37,618 - INFO - duckdb.py:414 - Persisting DB to disk, putting it in the save folder: /Users/anurag.thakur/llama/localGPT/DB

**Comment by RealLau** [comment]
> Yes it took me 7 mins
> 
> ```
> 2023-07-29 16:06:00,887 - INFO - __init__.py:88 - Running Chroma using direct local API.
> 2023-07-29 16:06:12,145 - WARNING - __init__.py:43 - Using embedded DuckDB with persistence: data will be stored in: /Users/anurag.thakur/llama/localGPT/DB
> 2023-07-29 16:06:13,775 - INFO - ctypes.py:22 - Successfully imported ClickHouse Connect C data optimizations
> 2023-07-29 16:06:14,198 - INFO - json_impl.py:45 - Using python library for writing JSON byte strings
> 2023-07-29 16:06:14,874 - INFO - duckdb.py:454 - No existing DB found in /Users/anurag.thakur/llama/localGPT/DB, skipping load
> 2023-07-29 16:06:14,874 - INFO - duckdb.py:466 - No existing DB found in /Users/anurag.thakur/llama/localGPT/DB, skipping load
> 
> 
> 2023-07-29 16:12:37,512 - INFO - duckdb.py:414 - Persisting DB to disk, putting it in the save folder: /Users/anurag.thakur/llama/localGPT/DB
> 2023-07-29 **16:12:37,618** - INFO - duckdb.py:414 - Persisting DB to disk, putting it in the save folder: /Users/anurag.thakur/llama/localGPT/DB
> ```

Let's add a tip to let us know it's still working...
